ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueSubmission.scala

```diff
@@ -35,7 +35,10 @@ object KeyValueSubmission {
     *
     * @deprecated Use [[KeyValueCommitting.submissionOutputs]] instead. This function will be removed in later version.
     */
-  def transactionOutputs(entryId: DamlLogEntryId, tx: SubmittedTransaction): List[DamlStateKey] = {
+  def transactionOutputs(
+      entryId: DamlLogEntryId,
```

Comment:
Do we need `entryId` anymore? Preferrably we remove the whole `submissionOutputs` function and include the outputs in the submission message itself. This can and should probably be later work.

Reply:
Does not seem with need it.

Reply:
dropped